go.mod

```diff
@@ -167,3 +168,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/sigstore/protobuf-specs => /Users/tetsuo/Code/protobuf-specs
```

Comment:
If you want to test this locally, clone this branch: https://github.com/trail-of-forks/protobuf-specs/tree/alex/configurable-crypto
And point it to your local copy.